www-servers/sniproxy/sniproxy-0.5.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools user
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/dlundquist/sniproxy.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/dlundquist/sniproxy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Proxies incoming HTTP and TLS connections based on the hostname"
+HOMEPAGE="https://github.com/dlundquist/sniproxy"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="+dns +largefile rfc3339"
+
+RDEPEND="
+	dev-libs/libev
+	>=dev-libs/libpcre-3
+	dns? ( net-libs/udns )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+
+	{
```

Comment:
Why are you trying hard to make things overcomplex? If you need to call `sed` two times, just call it two times and `||die` separately so that the PM can report exact failing line instead of 'one of N commands failed'.